github_issue_summarization/03_serving_the_model.md

```diff
@@ -83,12 +101,25 @@ Now that you have seldon core deployed, you can deploy the model using the
 ```bash
 ks generate seldon-serve-simple-v1alpha2 issue-summarization-model \
```

Comment:
Should we just check in the resulting ksonnet module into the ksonnet app
https://github.com/kubeflow/examples/tree/master/github_issue_summarization/ks_app

And then tell users to just add an environment and then deploy the component?

(We can also address that in a follow on PR).